homeassistant/components/nibe_heatpump/strings.json

```diff
@@ -2,8 +2,27 @@
   "config": {
     "step": {
       "user": {
+        "menu_options": {
+          "nibegw": "NibeGW",
+          "modbus": "Modbus"
+        },
+        "description": "Pick the connection method to your pump. In general F-series pumps require a Nibe GW custom accessory, while a S-series pump have Modbus support built in."
+      },
+      "modbus": {
+        "data": {
+          "model": "Model of Heat Pump",
+          "modbus_url": "Modbus URL",
+          "modbus_unit": "Modbus Unit Identifier"
+        },
+        "data_description": {
+          "modbus_url": "Modbus URL that describes the connection to your Heat Pump or MODBUS40 unit. It should be on the form tcp://[HOST]:[PORT] for Modbus TCP connection, serial://[LOCAL DEVICE] for a local Modbus RTU connection, rfc2217://[HOST]:[PORT] for a remote telnet based Modbus RTU connection.",
```

Comment:
Not sure if it would be better or not but it might be more readable if you code fence `tcp://[HOST]:[PORT]` etc

Reply:
I split them over rows and fenced them. A little bit more readable.